docs/modules/ROOT/pages/contributor-guide/index.adoc

```diff
@@ -0,0 +1,47 @@
+[[contributor-guide]]
+= Contributor guide
+
+
+[[prerequisites]]
+== Prerequisites
+
+* `git`
+* GraalVM with `native-image` command installed and `GRAALVM_HOME` environment variable set, see
+  https://quarkus.io/guides/building-native-image-guide[Building a native executable] section of the Quarkus
+  documentation.
+* If your are on Linux, `docker` is sufficient for the native mode too. Use `-Pnative,docker` instead of `-Pnative`
+  if you choose this option.
+* Java 11
+* Maven (`mvn`), Maven Daemon (`mvnd` for fast builds), or Maven Wrapper (`mvnw`)
+
+[[how-to-build]]
+== How to build
+
+Checkout the code
+
+[source,shell]
+----
+$ git clone https://github.com/quarkiverse/quarkus-cxf.git -o upstream
+$ cd camel-quarkus
```

Comment:
```suggestion
$ cd quarkus-cxf
```
...no?

Reply:
Thanks, of course, a copy-and-paste error